csun-metalab/nomi
# NAM-316 -- UI: Update Roster List View

# Description
Set class container to full and added standard containers to studentmatrix.
  
# Testing
1. Ensure the list view extends the full screen width without going out of bounds.
2. Ensure all other views are properly set.
3. Ensure padding and margin fit the criteria of the task.
